app/assets/stylesheets/theme/panels.scss

```diff
@@ -21,6 +21,10 @@ $brave-panels-borderRadius: 8px;
       min-height: $spacer * 10;
     }
 
+    &--short {
```

Comment:
The CSS for the box looks a little odd because of some of the padding issues, might wanna work with Ross to see if there is an easy solution with regards to the CSS

Reply:
From the screenshot it looks to me like there might be something causing a little too much vertical space to be added. Doesn't seem to be min-height thougt cause it's happening on the large panel too. Might help to just even that spacing out a bit.